test/verify/check-client

```diff
@@ -37,14 +37,20 @@ class TestClient(testlib.MachineCase):
         self.m_target.execute("hostnamectl set-hostname target")
         # validate on-demand install: this does not work on arch, non-split package
         if self.m_target.image.startswith("debian") or self.m_target.image.startswith("ubuntu"):
-            self.m_target.execute("dpkg --purge cockpit-pcp-dbgsym || true; dpkg --purge cockpit-pcp pcp")
+            self.m_target.execute("""
+                if dpkg -l python3-pcp; then
```

Comment:
When would that ever not be true? If pcp depends on python3-pcp, and the else branch removes pcp, then python3-pcp should always be installed, too.

Reply:
It's a fallback as we afaik have no python3-pcp on debian-testing:

```
root@debian-testing-127-0-0-2-2201:~# dpkg -l | grep python3-pcp
root@debian-testing-127-0-0-2-2201:~#
```

Reply:
But yeah it should also remove pcp...

Reply:
IMHO it would make things a lot simpler to just deal with the `pcp` package and nothing else. Here in the test, in the packaging dependencies, and in the on-demand install. The latter already tests for the `.service` (which is pcp, not python3-pcp). That would make everything consistent and simpler.